app/tests/infrastructure/monitoring/test_metrics.py

```
@@ -0,0 +1,160 @@
+    track_vault_pin_set,
+    track_vault_member_added,
+    track_pin_lockout,
+    track_auth_failure,
+)
+
+
+def _get_counter_value(counter, **labels) -> float:
+    """Helper to get current counter value for given labels."""
+    try:
+        return counter.labels(**labels)._value.get()
+    except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)